README.md

```diff
@@ -143,10 +143,10 @@ func main() {
 
 type Post struct {
     // db tag lets you specify the column name if it differs from the struct field
-    Id      int64 `db:"post_id"`
+    Id      int64  `db:"post_id"`
     Created int64
-    Title   string
-    Body    string
+    Title   string `db:",50"`       // Column size set to 50
+    Body    string `db:"text,1024"` // Set both column name and size
```

Comment:
Looking at the parsing code, shouldn't these tags be `db:",size:50` and `db:"text,size:1024"`, respectively?